build.gradle

```diff
@@ -119,6 +119,7 @@ allprojects {
 }
 
 repositories {
+    mavenLocal()
```

Comment:
I know it is required for local development, but ideally we should not put this, as it can lead to errors with different people can different versions. 

These should be added as a part of Development.md file that we need to enable mavenLocal(). But good for now.